uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/data/leave-request-data.js

```diff
@@ -111,6 +217,48 @@ define(function () {
     },
     balanceChangeByAbsenceType: function () {
       return mockData.balanceChangeByAbsenceTypeData;
+    },
+    getRandomLeaveRequest: function () {
+      return mockData.randomLeaveRequestFromServer;
+    },
+    createRandomLeaveRequest: function () {
+      return mockData.randomLeaveRequestToServer;
+    },
+    updateRandomLeaveRequest: function () {
+      return mockData.randomUpdateLeaveRequestToServer;
+    },
+    calculateBalanceChangeRequest: function () {
+      return mockData.calculateBalanceChangeRequestData;
+    },
+    calculateBalanceChange: function () {
+      return mockData.calculateBalanceChangeData;
+    },
+    getAllRequestDayTypes: function () {
+      return optionGroupMock.getAllValuesForKey('hrleaveandabsences_leave_request_day_type', 'name');
+    },
+    getAllRequestDayValues: function () {
+      return optionGroupMock.getAllValuesForKey('hrleaveandabsences_leave_request_day_type', 'value');
+    },
+    getAllRequestStatusesNames: function () {
+      return optionGroupMock.getAllValuesForKey('hrleaveandabsences_leave_request_status', 'name');
+    },
+    getAllRequestStatusesLabels: function () {
+      return optionGroupMock.getAllValuesForKey('hrleaveandabsences_leave_request_status', 'label');
+    },
+    getAllRequestStatusesValues: function () {
+      return optionGroupMock.getAllValuesForKey('hrleaveandabsences_leave_request_status', 'value');
+    },
+    getAllAbsenceTypesTitles: function () {
+      return absenceTypeData.getAllAbsenceTypesByKey('title');
+    },
+    getAllAbsenceTypesIds: function () {
```

Comment:
Why are all these functions (`getAllRequestDayTypes`, `getAllRequestDayValues`, `getAllRequestStatusesNames`, `getAllRequestStatusesLabels`, `getAllRequestStatusesLabels`, `getAllRequestStatusesValues`, `getAllAbsenceTypesTitles`,  `getAllAbsenceTypesIds`)  here, in the leave request mock data file, if they are about option groups?